keras_cv/layers/preprocessing/with_mixed_precision_test.py

```diff
@@ -86,20 +97,54 @@
     (
         "RandomGaussianBlur",
         layers.RandomGaussianBlur,
-        {"kernel_size": 3, "factor": (0.0, 3.0)},
+        {"kernel_size": 3, "factor": (0.0, 3.0), "dtype": "float32"},
+    ),
+    (
+        "RandomJpegQuality",
+        layers.RandomJpegQuality,
+        {"factor": (75, 100), "dtype": "float32"},
+    ),
+    (
+        "RandomRotation",
+        layers.RandomRotation,
+        {
+            "factor": 0.5,
+            "bounding_box_format": "xyxy",
+            "dtype": "float32",
+        },
     ),
-    ("RandomJpegQuality", layers.RandomJpegQuality, {"factor": (75, 100)}),
-    ("RandomRotation", layers.RandomRotation, {"factor": 0.5}),
     ("RandomSaturation", layers.RandomSaturation, {"factor": 0.5}),
     (
         "RandomSharpness",
         layers.RandomSharpness,
         {"factor": 0.5, "value_range": (0, 255)},
     ),
-    ("RandomAspectRatio", layers.RandomAspectRatio, {"factor": (0.9, 1.1)}),
-    ("RandomShear", layers.RandomShear, {"x_factor": 0.3, "x_factor": 0.3}),
+    (
+        "RandomAspectRatio",
+        layers.RandomAspectRatio,
+        {
+            "factor": (0.9, 1.1),
+            "bounding_box_format": "xyxy",
+            "dtype": "float32",
+        },
+    ),
+    (
+        "RandomShear",
+        layers.RandomShear,
+        {
+            "x_factor": 0.3,
+            "x_factor": 0.3,
+            "bounding_box_format": "xyxy",
+            "dtype": "float32",
+        },
+    ),
+    # ("RandomShear", layers.RandomShear, {"x_factor": 0.3, "x_factor": 0.3}),
```

Comment:
Let's either un-comment this or re-add it. Not sure when this originally got commented-out

Reply:
This is weird... the test above it is the same one. I think I may have added it when fixing some styling issues. I'll just remove it to avoid duplicates.